src/MetaModels/FrontendIntegration/MetaModelHybrid.php

```diff
@@ -96,11 +97,41 @@ public function __construct($objElement, $strColumn = 'main')
     public function generate()
     {
         if (TL_MODE == 'BE') {
+
+            $strInfo = '';
+            if ($this->metamodel) {
+                // Retrieve Name of MetaModels
+                $factory       = $this->getServiceContainer()->getFactory();
+                $metaModelName = $factory->translateIdToMetaModelName($this->metamodel);
+                $metaModel     = $factory->getMetaModel($metaModelName);
+                $strInfo       = '<br><span style="width:31px;display:inline-block;">MM: </span>' . $metaModel->getName();
+
+                $database = \Database::getInstance();
+
+                // Retrieve Name of Filter.
+                if ($this->metamodel_filtering) {
+                    $infoFi = $database
+                        ->prepare('SELECT * FROM tl_metamodel_filter WHERE id=?')
+                        ->execute($this->metamodel_filtering)
+                        ->row();
+                    $strInfo .= '<br><span style="width:31px;display:inline-block;">Fi: </span>' . $infoFi['name'];
```

Comment:
Same as above:
- Styles should be moved to an extra css file.
- Please use the `abbr` tag and provide a title.